apps/question-bank/src/main/java/gov/cdc/nbs/questionbank/question/request/update/UpdateDateQuestionRequest.java

```diff
@@ -1,5 +1,6 @@
 package gov.cdc.nbs.questionbank.question.request.update;
 
+import gov.cdc.nbs.questionbank.question.request.create.CreateDateQuestionRequest.DateMask;
```

Comment:
question (non-blocking): What is `DateMask`?  If there is only one is it just a boolean?